### PR TITLE
Files for Backend-Last-Byte-Response time

### PR DESCRIPTION
**Pull request by jaesoni**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [/] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@jaesoni - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@jaesoni - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@jaesoni - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: demos/ag-alert-backend-lastbyte-resp/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+This template deploys an Azure Monitor alert which notifies you in an event **the average backend last byte response time is greater than [dynamic threshold](https://docs.microsoft.com/en-us/azure/azure-monitor/alerts/alerts-dynamic-thresholds)**.  
+
+**Notes**
+
+- Azure Monitor alert rules are charged based on the type and number of signals it monitors. You may want to visit the [pricing page](https://azure.microsoft.com/en-in/pricing/details/monitor/) before deploying this alert template or can view the estimated cost after the deployment. 
+
+- You will need to create [Azure Monitor Action Group](https://docs.microsoft.com/en-us/azure/azure-monitor/alerts/action-groups) in advance and provide its ResourceID during this deployment. This action group notifies your users when the alert rule is triggered. You can use an existing or create a new one and reuse it for multiple such alerts.
+
+>[!TIP]
```
Comment (bmoore-msft) [inline demos/ag-alert-backend-lastbyte-resp/README.md:24]: this is not valid markdown for github

File: demos/ag-alert-backend-lastbyte-resp/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
+{
+   "$schema":"https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+   "contentVersion":"1.0.0.0",
+   "parameters":{
+      "alertName":{
+         "type":"String"
+      },
+      "applicationGatewayResourceID":{
```
Comment (bmoore-msft) [inline demos/ag-alert-backend-lastbyte-resp/azuredeploy.json:8]: make the same changes here as in #12205

File: demos/ag-alert-backend-lastbyte-resp/metadata.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "itemDisplayName": "Create an alert using Backend Last Byte Response metric for your Application Gateway.",
```
Comment (bmoore-msft) [inline demos/ag-alert-backend-lastbyte-resp/metadata.json:3]: see pipeline result for changes that need to be made here

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: demos/ag-alert-backend-lastbyte-resp/README.md (hunk 1 above)
Comment (jaesoni) [reply demos/ag-alert-backend-lastbyte-resp/README.md:24]: @bmoore-msft - Can you please specifiy which part exactly is in violation?

**Comment by azure-quickstarts** [comment]
@jaesoni - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: demos/ag-alert-backend-lastbyte-resp/azuredeploy.json (hunk 2 above)
Comment (jaesoni) [reply demos/ag-alert-backend-lastbyte-resp/azuredeploy.json:8]: https://github.com/Azure/azure-quickstart-templates/pull/12205#discussion_r812779879

File: demos/ag-alert-backend-lastbyte-resp/metadata.json (hunk 3 above)
Comment (jaesoni) [reply demos/ag-alert-backend-lastbyte-resp/metadata.json]: done

File: demos/ag-alert-backend-lastbyte-resp/README.md (hunk 1 above)
Comment (bmoore-msft) [reply demos/ag-alert-backend-lastbyte-resp/README.md:24]: ```[!TIP]``` is for docs.microsoft.com - github renders it literally.

Comment (jaesoni) [reply demos/ag-alert-backend-lastbyte-resp/README.md]: Updated the file

**Comment by azure-quickstarts** [comment]
@jaesoni - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Pull request merged**